### PR TITLE
firebase-app-distribution 0.0.1

### PR DESCRIPTION
**Pull request by guness**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by lszucs** [comment]
> PR with such step already exists: #2151

Unfortunately, due to Firebase App Distro not being public at the time, we could not disclose, that we've been working together with a contributor on this step earlier.

As stated on the linked PR, we decided to go with this PR because of the following reasons:
- Step test CI is set up for both platforms
- Step does input validation
- Step employs our usual step practices, such as config printing, colored logging

File: steps/firebase-app-distribution/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+title: '[Alpha]Firebase App Distribution'
+summary: |
+  [Alpha]Distributes your builds via Firebase App Distribution
+description: |
+  [Alpha]Upload your app binaries to Firebase and sends email to your testers.
```
Comment (koral--) [inline steps/firebase-app-distribution/0.0.1/step.yml]:
```suggestion
  [Alpha]Uploads your app binaries to Firebase and sends email to your testers.
```

File: steps/firebase-app-distribution/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- firebase_token: null
+  opts:
+    description: |
+      Firebase CI access token. You can create by `firebase login:ci` on your local machine.
```
Comment (koral--) [inline steps/firebase-app-distribution/0.0.1/step.yml]:
```suggestion
      Firebase CI access token. You can create it by `firebase login:ci` on your local machine.
```

File: steps/firebase-app-distribution/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+      It is something in the form of `1/a1gqPSI3ryNcsDiZnCaHnY-hg9u9nJeosz3iNPe-S-RLdmqdypiJMOjoeOhOGsWK`
+    is_required: true
+    is_sensitive: true
+    title: Firebase Token
+- app_path: $BITRISE_IPA_PATH|$BITRISE_APK_PATH
+  opts:
+    description: |-
+      Path for IPA or APK file to deploy.
```
Comment (koral--) [inline steps/firebase-app-distribution/0.0.1/step.yml]:
```suggestion
      Path of IPA or APK file to deploy.
```

**Comment by guness** [comment]
closing this PR in favor of version 0.1.0, recommended changes applied.

**Pull request closed without merging**